pmix.sty

```diff
@@ -215,6 +215,7 @@
 \newcommand{\declareconstitem}[1]{\item[\code{#1}] \index{#1} \hspace{1em}}
 \newcommand{\declareconstitemvalue}[2]{\item[\code{#1}] \index{#1} \hspace{0.25em} \code{#2}  \hspace{1em}}
 \newcommand{\declareconstitemDEP}[2]{\item[\code{#1} (Deprecated in PMIx #2)] \index{#1} \hspace{1em}}
+\newcommand{\declareconstitemNEW}[1]{\item[\color{magenta}\code{#1}] \index{#1} \hspace{1em}}
```

Comment:
I don't mind a 'new' highlight, which will move back to normal in the next release. Should we add the version marker (`\versionMarker{3.0}`) at the call locations to make sure they are labeled with the version they first appear in?

Reply:
I tried doing that first, but the marker is slightly shifted from the actual line and it gets confusing since each constant is basically one line. If we can get them better aligned, then that would indeed be an option.